tvb_contrib/tvb/contrib/tests/cosimulation/parallel/double_proxy_precision_complex_delay_update_test.py

```diff
@@ -36,8 +36,8 @@ class TestDoubleProxyPrecisionComplexDelayUpdate(BaseTestCase):
     """
 
     def test_double_precision_complex_delay_update(self):
-        weight = np.array([[5, 2, 4, 0], [8, 5, 4, 1], [6, 1, 7, 9], [10, 0, 5, 6]], dtype=np.float_)
-        delay = np.array([[7, 8, 5, 1], [9, 3, 7, 9], [4, 3, 2, 8], [9, 10, 11, 5]], dtype=np.float_)
+        weight = np.array([[5, 2, 4, 0], [8, 5, 4, 1], [6, 1, 7, 9], [10, 0, 5, 6]], dtype=np.float64)
+        delay = np.array([[7, 8, 5, 1], [9, 3, 7, 9], [4, 3, 2, 8], [9, 10, 11, 5]], dtype=np.float64)
```

Comment:
why only here needs to be changed?

I see other places where `np.float_` is still used

Reply:
thanks for the observation, I will update them everywhere